zh_cn.lang

```diff
@@ -415,98 +415,119 @@ gui.xaero_category_enable_protection=保护：关
 gui.xaero_category_disable_protection=保护：开
 gui.xaero_category_disable_protection_confirm=禁用以下类别的保护？
 gui.xaero_category_enable_protection_confirm=启用以下类别的保护？
-gui.xaero_box_category_protection=类别保护可以让你避免意外删除、移动、更名一个类别或改变该类别的“硬包含”设置。
-gui.xaero_category_protected_category=已保护
+gui.xaero_box_category_protection=类别保护可防止你（意外）删除、移动、重命名类别或更改其“硬包含”设置。
+gui.xaero_category_protected_category=受保护！
 gui.xaero_category_category_move_up=提高优先级
 gui.xaero_category_category_move_down=降低优先级
 gui.xaero_category_delete_list_element=删除
 gui.xaero_category_settings_confirm=确认
 gui.xaero_category_settings_cancel=取消
-gui.xaero_category_settings_cancel_confirm=放弃更改并直接离开？
+gui.xaero_category_settings_cancel_confirm=离开而不保存你的更改？
 gui.xaero_category_settings_reset=重置为默认
-gui.xaero_category_settings_reset_confirm1=重置为默认的类别配置？
-gui.xaero_category_settings_reset_confirm2=自定义的实体类别会被移除，默认的实体类别会被重置。
+gui.xaero_category_settings_reset_confirm1=将类别配置重置为默认？
+gui.xaero_category_settings_reset_confirm2=自定义类别将被删除，默认类别将被重置。
 gui.xaero_radar_displayed=显示
 gui.xaero_radar_dots_color=点颜色
 gui.xaero_radar_icons_displayed=图标
 gui.xaero_radar_names_displayed=名称
-gui.xaero_radar_render_order=渲染次序
-gui.xaero_box_radar_render_order=配置雷达类别渲染的次序。数字越低越早渲染。但是，在新版的Minecraft中，点始终渲染在图标上方，但此顺序仍可应用于点或图标的渲染。
+gui.xaero_radar_render_order=渲染顺序
+gui.xaero_box_radar_render_order=控制雷达类别的渲染顺序。较低的数字意味着更早地渲染。 然而，在较新的Minecraft版本中，点总是显示在图标之上。 该顺序仍然适用于点或图标渲染。
 gui.xaero_radar_dots_color_team_colour=队伍颜色
-gui.xaero_box_always_entity_nametags2=Always render names for entities with name tags regardless of what the name setting is set to.
+gui.xaero_box_always_entity_nametags2=无论名称设置是什么，始终为带有命名牌的实体显示名称。
 gui.xaero_radar_setting_displayed=显示雷达
 gui.xaero_radar_setting_names=显示名称
 gui.xaero_radar_setting_icons=显示图标
-gui.xaero_toggle_entity_radar=实体雷达开关
-narration.xaero_ui_list_left_right_usage=Press LEFT and RIGHT buttons to select a column within the entry.
-gui.xaero_category_help2=选择一个子类别，并前往其设置以配置点颜色、名称、图标等设置。\n \n 多个子类别包含同一实体时，优先选择列表中更高的类别（创建自定义类别时要记住这点）。通常，新的自定义类别应该放在最上面。
+gui.xaero_toggle_entity_radar=切换实体雷达
+narration.xaero_ui_list_left_right_usage=按下“左”和“右”按钮选择条目中的一列。
```

Comment:
Translates back weird. Does this work for "a column within the entry", not plural "entries"?